astropy/modeling/functional_models.py

```diff
@@ -281,6 +298,24 @@ def __init__(self, amplitude=amplitude.default, x_mean=x_mean.default,
             amplitude=amplitude, x_mean=x_mean, y_mean=y_mean,
             x_stddev=x_stddev, y_stddev=y_stddev, theta=theta, **kwargs)
 
+    def _bounding_box_default(self):
+        """
+        Returns the default `bounding_box` limits for a `Gaussian2D` instance.
+
+        The default for `Gaussian2D` is :math:`\pm 5.5\sigma` relative to the mean,  
+        and adjusted for rotation. 
+        """
+
+        from . utils import ellipse_extent
+
+        a = 5.5 * self.x_stddev
+        b = 5.5 * self.y_stddev
```

Comment:
I think it would be great if this value of `5.5` could be adjusted by the user, but I don't have a good suggestion how to handle this. Anyway it can be addressed later and shouldn't stop this PR...